codingStandards/CodingStandard-Html.md

```diff
@@ -237,3 +237,33 @@
       </td>
     </tr>
   </table>
+
+- Always remember to include `rel="noopener noreferrer"` when using `target="_blank"` in links. Leaving this out would enable attackers to get hold of `window.opener` object, which, for example, can used to redirect to a malicious link. [\[Source\]](https://medium.com/@jitbit/target-blank-the-most-underestimated-vulnerability-ever-96e328301f4c#.jlkn46vtk)
```

Comment:
While you are at it, give the rationale as a block quote? We use block quotes for additional info that can be skipped.